src/index.tsx

```
@@ -1,13 +1,33 @@
+import * as React from 'react';
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { HomePage } from './pages/HomePage';
+import { NotFoundPage } from './pages/NotFoundPage';
+import { PeoplePage } from './pages/PeoplePage';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
```

Minor suggestion: you assert the root element with `as HTMLDivElement` in the createRoot call. Consider guarding against a null result from `document.getElementById('root')` and giving a clearer error if missing (e.g., throw a helpful message) so the app fails fast with an understandable error instead of relying on a type assertion.